.github/workflows/ci.yml

```diff
@@ -38,6 +38,7 @@ jobs:
             php-version: '7.4'
             job-description: 'with migration rules'
             execute-migration-rules: 'yes' # should be checked on highest supported PHP version
+            phpunit-coverage: '--coverage-clover=./build/logs/clover.xml'
```

Comment:
pls, run it out of job `with migration rules`, as code files with rules applied is different